app/views/layout.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Chitter</title>
+  </head>
+  <body>
```

Comment:
You asked about getting the CSS working with a div, but I didn't see any CSS in the pull request.  Grab me if you want to chat about it in person!